pjmedia/include/pjmedia/wav_port.h

```diff
@@ -128,6 +128,10 @@ PJ_DECL(pj_ssize_t) pjmedia_wav_player_get_len(pjmedia_port *port);
 
 /**
  * Set the file play position of WAV player.
+ * @b Safety: This function can only be called when the player is stopped
```

Comment:
I prefer the term "not running", instead of "stopped", because the latter implies that we need to explicitly stop the player first. While actually as long as it's not running, it should be okay, such as during creation.

Reply:
Agreed and updated